### DISCUSSION TITLE
How to display all content in one page only

### DISCUSSION THREAD

**Discussion by linhltAlobridge** [dompdf/dompdf#2840] [Q&A]
Hi all,

I'm quite new to the programming world and have just started out so sorry if i don't explain this correctly.

I am using dompdf to create CV. CV page is longer than 2 a4 page size but what i want is to print cv pdf file only in a single page. so how can i do that?

**Comment by bsweeney** [comment]
You can specify a custom page size to accommodate the whole document on a single page. For example, the following is the same width but twice the height as a4 (595.28pt x 841.89pt).

```
$dompdf->setPaper([0, 0, 595.28, 1683.78]);
```
